index.js

```diff
@@ -129,9 +146,15 @@ function getCarInfoById(/* code here */) {
  * sortCarInventory takes a single argument:
  *     (1) an array which is an inventory of cars like the one inside /data/inventory.js.
  * sortCarInventory returns an inventory that is sorted by car_model, ascending [A-Z].
-*/
-function sortCarInventory(/* code here */) {
-  /* code here */
+ */
+function sortCarInventory(inventory) {
+    return inventory.sort((a, b) => {
+        let x = a.car_model.toLowerCase();
+        let y = b.car_model.toLowerCase();
+        if (x < y) { return -1; }
+        if (x > y) { return 1; }
+        return 0;
+    });
 }
```

Comment:
I like how you organized this problem.  However i would like to have seen you use 'else' instead of the second if statement.

Reply:
this callback return 3 different state: -1 if x < y; 1 if x > y; and 0 if x === y. Normaly I should have used else if statement but  I wanted to code less.